ci/scripts/check_latest_release_is_published.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
+#
+# Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+# This product includes software developed at Datadog (https://www.datadoghq.com/).
+# Copyright 2016-Present Datadog, Inc.
+#
+
+set -o pipefail
+
+tag_name=$(curl -s -H "Authorization: Bearer $GITHUB_TOKEN" \
+     -H "Accept: application/vnd.github+json" \
+     https://api.github.com/repos/DataDog/dd-sdk-android/releases/latest | jq -r .tag_name)
+
+repos_list=(
```

Comment:
suggestion: maybe we can think of building it dynamically. Essentially, all folders starting with `dd-sdk-android-` represent published modules, at least it is true as of now.